src/main/java/uk/gov/companieshouse/items/orders/api/controller/CertificateItemsController.java

```diff
@@ -70,4 +65,18 @@ public CertificateItemsController(final CreateItemRequestValidator validator,
         return ResponseEntity.status(HttpStatus.CREATED).body(createdCertificateItemDTO);
     }
 
+    @GetMapping("/certificates/{certificate_id}")
+    public ResponseEntity<Object> getCertificateItem(@PathVariable String certificate_id)
```

Comment:
`certificate_id`: I don't think we use snake case much for Java variable names. Typically the names are  camel case.

Reply:
I only used snake case, because that's how it looked here https://developer-specs.kermit.aws.chdev.org/order-company-products/reference/certificates/get-a-certificate-resource. But you are right, we don't use it anywhere else, so I will change it to camel case.